_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -5953,6 +5955,7 @@ void MfxHwH264Encode::SetDefaults(
             par.calcParam.mvcPerViewPar.bufferSizeInKB = !IsHRDBasedBRCMethod(par.mfx.RateControlMethod)
                     ? GetMaxCodedFrameSizeInKB(par)
                     : bufferSizeInBits / 8000;
+            par.calcParam.mvcPerViewPar.bufferSizeInKB = MFX_MIN(par.calcParam.mvcPerViewPar.bufferSizeInKB, par.calcParam.mvcPerViewPar.initialDelayInKB);
```

Comment:
same: why min?

Reply:
see description in JIRA

Reply:
What the Jira?
Not clear what this code does.
Imagine bufferSizeInKB is < than initialDelayInKB.
In the result still bufferSizeInKB < initialDelayInKB! Didn't you care about it at 2697.
If you are sure that bufferSizeInKB >= initialDelayInKB, then simply assign... without computations.

Reply:
Leonid, please take a look at problem description. I copied to comment special for you:
Issue: 49068
Test: manually (beh test is needed)
If BufferSizeInKB==0 && InitialDelayInKB!=0
then BufferSizeInKB should be set >= InitialDelayInKB

Reply:
Then max, not min.

Reply:
Ops, my typo. I replaced IPP_MAX on MFX_MIN in patch 2. Thanks.